requirements_dev.txt

```diff
@@ -1,142 +1,8 @@
-absl-py==1.4.0
-anyio==3.7.0
-appnope==0.1.3
-argon2-cffi==21.3.0
-argon2-cffi-bindings==21.2.0
-arrow==1.2.3
-asttokens==2.2.1
-astunparse==1.6.3
-async-lru==2.0.2
-attrs==23.1.0
-Babel==2.12.1
-backcall==0.2.0
-beautifulsoup4==4.12.2
-bleach==6.0.0
-cachetools==5.3.0
-certifi==2023.5.7
-cffi==1.15.1
-charset-normalizer==3.1.0
-cloudpickle==2.2.1
-comm==0.1.3
-contourpy==1.0.7
-cycler==0.11.0
-debugpy==1.6.7
-decorator==5.1.1
-defusedxml==0.7.1
-dm-tree==0.1.8
-exceptiongroup==1.1.1
-executing==1.2.0
-fastjsonschema==2.17.1
-flatbuffers==23.5.9
-fonttools==4.39.4
-fqdn==1.5.1
-gast==0.4.0
-google-auth==2.18.0
-google-auth-oauthlib==1.0.0
-google-pasta==0.2.0
-grpcio==1.54.2
-h5py==3.8.0
-idna==3.4
-iniconfig==2.0.0
-ipykernel==6.23.3
-ipython==8.14.0
-ipython-genutils==0.2.0
-isoduration==20.11.0
-jax==0.4.10
-jedi==0.18.2
-Jinja2==3.1.2
-json5==0.9.14
-jsonpointer==2.4
-jsonschema==4.17.3
-jupyter-events==0.6.3
-jupyter-lsp==2.2.0
-jupyter_client==8.3.0
-jupyter_core==5.3.1
-jupyter_server==2.7.0
-jupyter_server_terminals==0.4.4
-jupyterlab==4.0.2
-jupyterlab-pygments==0.2.2
-jupyterlab_server==2.23.0
-keras==2.12.0
-kiwisolver==1.4.4
-libclang==16.0.0
-Markdown==3.4.3
-MarkupSafe==2.1.2
 matplotlib==3.7.1
```

Comment:
Are we plotting anything?

Reply:
Only in [notebooks](https://github.com/aniskhan25/hmsc-hpc/tree/main/notebooks).

Reply:
Ok, these were useful but turned out to be unused (at this stage at least), so shall be moved to outside of the package. Therefore, the dependence shall be removed as well.